I.write-tests/05-get-second-third-smallest/get-second-third.js

```diff
@@ -1,7 +1,10 @@
+
+
 module.exports = function (array) {
-    const newArray = array.slice()
-    newArray.sort(function (x, y) {
+    
+    array.sort(function (x, y) {
         return x > y;
     });
-    return [newArray[1], newArray[2]];
-};
\ No newline at end of file
+    return array[1], array[2];
```

Comment:
This change is not correct, using the comma operator means that only `array[2]` gets returned.